Makefile

```diff
@@ -97,6 +97,17 @@ save ""
 appendonly no
 endef
 
+define REDIS9_CONF
+daemonize yes
+protected-mode no
+port 6387
+pidfile /tmp/redis9.pid
+logfile /tmp/redis9.log
+save ""
+appendonly no
+maxmemory-policy allkeys-lfu
+endef
+
```

Comment:
Why create new Redis instance after all these commits?

Reply:
> Why create new Redis instance after all these commits?

My mistake.I should do it in the beginning.It is a better way to prevent configuration from affecting other test cases.(I think )